src/test/java/hw4/steps/ActionStep.java

```diff
@@ -0,0 +1,52 @@
+package hw4.steps;
+
+import hw4.users.LoginUser;
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+
+public class ActionStep extends AbstractStep{
+
+    public ActionStep(WebDriver driver) {
+        super(driver);
+    }
+
+    @Step("Login with {loginUser} user")
+    public void performLogin(LoginUser loginUser) {
+        homePage.performLogin(loginUser);
+    }
+
+    @Step("Open home page")
+    public void openHomePage() {
+        homePage.openPage();
+    }
+
+    @Step("Open different elements page")
+    public void openDifferentElementsPage() {
+        homePage.navToDifferentElementsPage();
```

Comment:
make this method parametrized. It should accept the name of page which can be opened from nav bar